### PR TITLE
Ship 17.0.2 on stable channel

### PR DESCRIPTION
**Pull request by tilosp**
Depends on https://github.com/nextcloud/updater_server/pull/279

### CODE REVIEW

File: generate-stackbrew-library.sh [hunk 1]
```diff
@@ -2,7 +2,7 @@
 set -Eeuo pipefail
 
 declare -A release_channel=(
-	[stable]='16.0.6'
+	[stable]='17.0.1'
```
Comment (J0WI) [inline generate-stackbrew-library.sh]:
```suggestion
	[stable]='17.0.2'
```

File: generate-stackbrew-library.sh [hunk 2]
```diff
@@ -2,7 +2,7 @@
...
 	[production]='16.0.6'
```
Comment (J0WI) [inline generate-stackbrew-library.sh]:
```suggestion
 	[production]='17.0.2'
```

Comment (tilosp) [reply generate-stackbrew-library.sh]:
```suggestion
 	[production]='16.0.7'
```
the production channel is still on 16

**Comment by tilosp** [comment]
Sure

**Pull request merged**